service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/EhrExtractStatusService.java

```diff
@@ -401,12 +401,14 @@ public void updateEhrExtractStatusAck(String conversationId, EhrReceivedAcknowle
         if (hasAcknowledgementExceededEightDays(conversationId, ack.getReceived())
             && hasEhrStatusReceivedAckWithUnexpectedConditionErrors(conversationId)) {
 
-            logger().warn("Received an ACK message with a conversation_id: {}, but it will be ignored", conversationId);
+            logger().warn("Received an ACK message with conversation_id: {}, "
+                          + "but it is being ignored because the EhrExtract has already been marked as a failure and 8 days have passed.",
+                          conversationId);
```

Comment:
```suggestion
            logger().warn("Received an ACK message with conversation_id: {}, "
                          + "but it is being ignored because the EhrExtract has already been marked as failed from not receiving an acknowledgement from the requester in time.",
                          conversationId);
```